### PR TITLE
Update zha.markdown about supported devices

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Update zha.markdown supported devices section, (related to https://github.com/home-assistant/core/pull/68921 which added  ZCLv7 and ZCLv8 support).

By the way, surprised that ZHA was not mentioned on Home Assistant 2022.4 release notes as [ZCLv7/ZCLv8 support](https://github.com/zigpy/zha-device-handlers/pull/872) is a big deal.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -404,7 +404,9 @@ To help resolve any kinks or compatibility problems, report bugs as issues with
...
+There is no official compatibility list of supported devices for the simple reason that practically all Zigbee devices that are fully compliant with the official standards should technically be compatible with this Zigbee implementation and work out-of-the-box with the ZHA integration. The official standards are set by the [Zigbee Alliance](https://zigbeealliance.org) in the [Zigbee Cluster Library Specification (ZCL) Revision 8](https://zigbeealliance.org/wp-content/uploads/2021/10/07-5123-08-Zigbee-Cluster-Library.pdf) and earlier revisions of their ZCL specifications.
```
Comment (frenck) [inline source/_integrations/zha.markdown:407]: What does this add to the average end-user?

The current text IMHO is already covering it, without adding unneeded implementation details.

Comment (Hedda) [reply source/_integrations/zha.markdown:407]: I think it be good to mention "Zigbee Cluster Library Specification (ZCL) R8" for reference now that ZHA is fully compliant. How about if I instead just slightly rewrite the existing sentence to mention ZCL R8 in the "ZHA exception and deviation handling" section instead in relation to describing [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)?

That would only slightly change this existing sentence there:

The ZHA implementation in Home Assistant relies on a library called "[ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)" to resolve issues with Zigbee devices that do not fully conform with the Zigbee standards. The few devices that deviate from the **Zigbee specifications set by the [Zigbee Alliance](https://zigbeealliance.org)** may therefore require proper bug reports with debug logs from users to assistant the developers in writing custom [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) for all of a device functions to work properly with the ZHA integration.

Into:

The ZHA implementation in Home Assistant relies on a library called "[ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers)" to resolve issues with Zigbee devices that do not fully conform with the Zigbee standards. The few devices that deviate from the **official standards set by the [Zigbee Alliance](https://zigbeealliance.org) in the [Zigbee Cluster Library Specification](https://zigbeealliance.org/wp-content/uploads/2021/10/07-5123-08-Zigbee-Cluster-Library.pdf) (revision 8 and earlier revisions of their ZCL specifications)** may therefore require proper bug reports with debug logs from users to assistant the developers in writing custom [ZHA Device Handlers](https://github.com/zigpy/zha-device-handlers/blob/dev/README.md) for all of a device functions to work properly with the ZHA integration.

Comment (frenck) [reply source/_integrations/zha.markdown:407]: > Zigbee Cluster Library Specification (ZCL) R8

No end-user cares about library specifications. There are jibberish terms for them and not relevant at all. They buy bulbs and want them to work.

Comment (Hedda) [reply source/_integrations/zha.markdown:407]: Closing PR

**Pull request closed without merging**